proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -206,6 +206,543 @@ function bufferNodeIs( test )
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty F64x';
+  var src = new F32x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /* BufferNode */
+
+  if( Config.interpreter === 'njs' )
+  {
+    test.case = 'check empty BufferNode';
+    var src = new BufferNode.alloc( 5 );
+    var got = _.bufferBytesIs( src );
+    var expected = false;
+    test.identical( got, expected );
+
+    test.case = 'check BufferNode';
+    var src = new BufferNode.allocUnsafe( 5, 4 );
+    var got = _.bufferBytesIs( src );
+    var expected = false;
+    test.identical( got, expected );
+
+    test.case = 'check BufferNode from string';
+    var src = new BufferNode.from( 'str' );
+    var got = _.bufferBytesIs( src );
+    var expected = false;
+    test.identical( got, expected );
+  }
+
+  /* BufferRaw */
+
+  test.case = 'check empty BufferRaw';
+  var src = new BufferRaw( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check BufferRawShared';
+  var src = new BufferRawShared( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /* BufferView */
+
+  test.case = 'check BufferView';
+  var src = new BufferView( new U8x( [ 1, 2, 3 ] ).buffer );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check without argument';
+  var got = _.bufferBytesIs();
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check extra arguments';
+  var got = _.bufferBytesIs( new U8x( [ 1, 2, 3 ] ), new BufferRaw( 5 ) );
+  var expected = true;
+  test.identical( got, expected );
+}
+
+//
+
+function constructorIsBuffer( test )
+{
+  test.case = 'check null';
+  var got = _.constructorIsBuffer( null );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check undefined';
+  var got = _.constructorIsBuffer( undefined );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check zero';
+  var got = _.constructorIsBuffer( 0 );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check false';
+  var got = _.constructorIsBuffer( false );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check NaN';
+  var got = _.constructorIsBuffer( NaN );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check Symbol';
+  var got = _.constructorIsBuffer( Symbol() );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty map';
+  var got = _.constructorIsBuffer( {} );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty array';
+  var got = _.constructorIsBuffer( [] );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check string';
```

Comment:
Please, add test cases with instances of BufferRaw, BufferView, BufferTyped, BufferNode
```js
var got = _.constructorIsBuffer( new BufferRaw( 3 ) );
```
```js
var got = _.constructorIsBuffer( new BufferView( new BufferRaw( 3 ) ) );
```
```js
var got = _.constructorIsBuffer( BufferNode.alloc( 3 ) );
```
```js
var got = _.constructorIsBuffer( new U8x( 3 ) );
```